src/test/scenarios/datafactory/output/src/datafactory/azext_datafactory/generated/_help.py

```diff
@@ -32,7 +32,7 @@
     examples:
       - name: Factories_Get
         text: |-
-               az datafactory show --name "exampleFactoryName" --resource-group "exampleResourceGroup"
+               az datafactory show --name "exampleFactoryName" --resource-group "myResourceGroup"
```

Comment:
I think what CLI team wanted is for all the resource name change it into myResourceName

Reply:
For ezample exampleFactoryName => myFactoryName

Reply:
internal resource name are fixed in here: https://github.com/Azure/autorest.az/pull/462/commits/9bf198e7499790b8f59b7bae84850f9ec6167acd